.github/workflows/agent-claude-review.yml

```
@@ -0,0 +1,112 @@
+      id-token: write
+      actions: read # Required for Claude to read CI results
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Run Claude Code Review
+        id: claude-review
+        if: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)